src/org/ods/util/Project.groovy

```diff
@@ -1034,13 +1034,17 @@ class Project {
 
     String toString() {
         // Don't serialize resolved Jira data items
-        def result = this.data.subMap(["build", "buildParams", "git", "jira", "metadata"])
+        def result = this.data.subMap(["build", "buildParams", "metadata", "git", "jira"])
 
+        if (!services?.jira && capabilities?.empty) {
+          result.remove("jira")
```

Comment:
@clemensutschig we should consider dumping data from Jira at least once (at the beginning) to help debug issues not shown or not handled during the pipeline run.